### PR TITLE
cleaner reading

### PR DESCRIPTION
**Pull request by joaquinelio**

### CODE REVIEW

**Comment by joaquinelio** [comment]
too many fragments, the () keeps it clean
im not sure about the other change.

not very important, the article is easy to understand if you followed the path

**Review by joachimklug — APPROVED** [review]

**Comment by iliakan** [comment]
This clause is essential to understanding, better to  leave it as is.

**Comment by joaquinelio** [comment]
I think it's a little hard to follow (too many commas) and it shouldn't be. 

But I didn't like my "fix" either.
